src/mpid/ch4/shm/posix/eager/iqueue/iqueue_init.c

```diff
@@ -24,12 +24,12 @@
     - name        : MPIR_CVAR_CH4_SHM_POSIX_IQUEUE_CELL_SIZE
       category    : CH4
       type        : int
-      default     : 69632
+      default     : 16384
       class       : none
       verbosity   : MPI_T_VERBOSITY_USER_BASIC
       scope       : MPI_T_SCOPE_ALL_EQ
       description : >-
-        Size of each cell. 4KB * 17 is default to avoid a cache aliasing issue.
```

Comment:
Hopefully the Intel folks can shed some light on this reasoning before we go forward with this.